api/tests/unit/environments/test_environments_views_sdk_environment.py

```diff
@@ -47,7 +47,7 @@ def test_get_environment_document(
     url = reverse("api-v1:environment-document")
 
     # When
-    with django_assert_num_queries(12):
+    with django_assert_num_queries(11):
```

Comment:
This seems like a good thing... but makes me nervous - do we know why we've dropped a query here?

Reply:
Yes we do — we weren't joining all of the integrations, see [this change](https://github.com/Flagsmith/flagsmith/blame/6c3ecc65be20792f06d6cd752e4e5d8d403fafe2/api/environments/managers.py#L17-L18).